http-server-netty/src/test/groovy/io/micronaut/docs/http/server/exception/SimpleControllerSpec.groovy

```diff
@@ -87,11 +88,25 @@ class SimpleControllerSpec extends Specification {
         }
 
         @Error
-        HttpResponse exceptionHandler(HttpRequest request, Throwable throwable) {
+        HttpResponse throwableExceptionHandler(HttpRequest request, Throwable throwable) {
+            JsonError error = new JsonError("Invalid: " + throwable.getMessage())
+            return HttpResponse.<JsonError>status(HttpStatus.BAD_REQUEST, "throwableExceptionHandler")
+                    .body(error)
+        }
+
+        @Error
+        HttpResponse exceptionHandler(HttpRequest request, RuntimeException throwable) {
             JsonError error = new JsonError("Invalid: " + throwable.getMessage())
             return HttpResponse.<JsonError>status(HttpStatus.BAD_REQUEST, "Fix it!")
                     .body(error)
         }
+
+        @Error
+        HttpResponse exceptionExceptionHandler(HttpRequest request, Exception throwable) {
+            JsonError error = new JsonError("Invalid: " + throwable.getMessage())
+            return HttpResponse.<JsonError>status(HttpStatus.BAD_REQUEST, "exceptionExceptionHandler")
+                    .body(error)
+        }
```

Comment:
What value is this method adding? Perhaps add a test that throws `Exception` to ensure the one that catches `Throwable` is not invoked?

Reply:
I was just trying to surround the error handler so it will trigger the problem because I’m not sure what is the logic to find the handler.